machinelearning/tensorflow.md

```diff
@@ -0,0 +1,349 @@
+# ML inference on Graviton CPUs with TensorFlow
+
+**Introduction**
+
+TensorFlow is an open-source software library for machine learning and artificial intelligence. It can be used across training and inference of deep neural networks. This document covers how to enable Tensorflow based machine learning inference on Graviton CPUs, what runtime configurations are important and how to debug any performance issues. The document also covers the instructions source builds to enable downstream features.
+
+# How to enable Tensorflow on Graviton CPUs
+To start with how to enable tensorflow inference on Graviton CPUs, there are multiple levels of software package abstractions available: Deep Lerning Amazon Machine Image (DLAMI), AWS Deep Learning Container (DLC), Docker hub container and Python wheel. Let's see how one can use each of them to get started.
```

Comment:
too complicated.. you can use it as a DLAMI, DLC, container or wheel. why would someone want the the docker hub contanier vs DLC? I don't see the DLC instructions below. Is the ordering right here? what is most popular/easiest? what is the first thing you want to show people?

Reply:
I agree. I cleaned up the aws DLC part in the below sections, but forgot to take it out from here. will fix. Regarding ordering..... I went with the ones that are more complete first, for example, DLAMI comes with all the packages pre-installed for bert based applications, and docker images come with some of the packages and benchmarks, and with wheel there is lot of flexibility, but one has to install all the dependent packages for ML applications. 
Please let me know if you think wheels should be first as they are more convenient for installing on the existing setup.